emission/analysis/modelling/tour_model/tour_model.md

```diff
@@ -0,0 +1,17 @@
+### Tour Model Guide
+
+In order to make a tour model from a user's e-mission location data, put all of the data in your mongodb, and then run 'python create_tour_model_matrix.py [UUID]'.
```

Comment:
This creates a tour model. How do I then generate new trips from the tour model using the simulation? How do I pass in the generated model instead of input.json?